pom.xml

```diff
@@ -112,7 +117,7 @@
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>workflow-basic-steps</artifactId>
-            <version>1.15</version>
+            <version>2.1-SNAPSHOT</version>
```

Comment:
:bug: this will be a cyclic dependency I think. That is why I purposefully kept tests specific to the `core` step in `CoreStepTest`.